cinema/views.py

```diff
@@ -1 +1,56 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.models import Movie, Actor, Genre, CinemaHall, MovieSession
+from cinema.serializers import (
+    ActorSerializer,
+    CinemaHallSerializer,
+    GenreSerializer,
+    MovieSerializer,
+    MovieListSerializer,
+    MovieSessionSerializer,
+    MovieSessionDetailSerializer,
+    MovieDetailSerializer,
+    MovieSessionListSerializer
+)
+
+
+class ActorViewSet(viewsets.ModelViewSet):
+    queryset = Actor.objects.all()
+    serializer_class = ActorSerializer
+
+
+class CinemaHallSet(viewsets.ModelViewSet):
+    queryset = CinemaHall.objects.all()
+    serializer_class = CinemaHallSerializer
+
+    def get_view_name(self):
+        if "list" in self.settings.VIEW_NAME_FUNCTION(self):
+            return "Cinema Halls"
+        return "Cinema Hall Instance"
+
+
+class GenreViewSet(viewsets.ModelViewSet):
+    queryset = Genre.objects.all()
+    serializer_class = GenreSerializer
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.prefetch_related("actors", "genres")
+
+    def get_serializer_class(self):
+        if self.action == "list":
+            return MovieListSerializer
+        if self.action == "retrieve":
+            return MovieDetailSerializer
+        return MovieSerializer
```

Comment:
![image](https://user-images.githubusercontent.com/107141441/230710952-8dc854f3-728d-4cc0-8948-d2dff022f67c.png)
I;m sorry, but where did we provide serializer_class attribute? Mb I'm missing the point, but it will return None in that case.